tests/e2e_tests/nonlin_e2e.py

```diff
@@ -140,6 +140,8 @@ def test_nonlin_cal_e2e(
     # We are going to make a new nonlinear calibration file using
     # a combination of the II&T nonlinearty file and the mock headers from
     # our unit test version of the NonLinearityCalibration
+    
```

Comment:
minor, but can we undo these two new lines if this PR isn't focused on this file?